docs/concepts/nodes.md

```diff
@@ -0,0 +1,55 @@
+---
+id: nodes
+title: Nodes
+sidebar_label: Nodes
+---
+
+## What is a node?
+
+NEAR Protocol runs on a collection of publicly maintained computers or "nodes". All nodes on a network perform and validate transactions, but differ in their ability to produce blocks.  
+
+## Node types
+
+For simplicities sake, NEAR has two main types of nodes: 
+  - block producing nodes (aka "validator nodes")
+  - non-block producing nodes (aka "regular nodes")
+
+### Non-block producing node ("regular node")
+
+Anyone can run a "regular node" by following one of the methods mentioned in [this guide](/docs/local-setup/running-a-node). 
+
+You may decide to run a node of your own for a few reasons:
+
+  - To view, process, and validate transactions on `MainNet`, `TestNet` or `BetaNet` (†)
```

Comment:
I think some reasons for running a regular node include:
1) serve rpc for your own use cases. This includes transaction submission, queries, view calls etc.
2) to index data from the network (running indexer on top of it for example)